go.mod

```diff
@@ -58,27 +58,29 @@ replace (
 )
 
 require (
-	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/Masterminds/semver/v3 v3.2.0
 	github.com/banzaicloud/logging-operator/pkg/sdk v0.8.16
 	github.com/banzaicloud/operator-tools v0.28.10
 	github.com/containernetworking/cni v1.1.2
+	github.com/docker/go-units v0.4.0
 	github.com/ehazlett/simplelog v0.0.0-20200226020431-d374894e92a4
 	github.com/emicklei/go-restful/v3 v3.8.0
 	github.com/gorilla/mux v1.8.0
 	github.com/guonaihong/gout v0.1.3
 	github.com/harvester/harvester-network-controller v0.3.1
+	github.com/harvester/node-manager v0.1.5-0.20230614075852-de2da3ef3aca
 	github.com/iancoleman/strcase v0.2.0
 	github.com/k3s-io/helm-controller v0.11.7
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v0.0.0-20200331171230-d50e42f2b669
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
 	github.com/kubernetes/dashboard v1.10.1
-	github.com/longhorn/backupstore v0.0.0-20230505043341-f871cdfdf2fc
-	github.com/longhorn/longhorn-manager v1.4.2
-	github.com/mattn/go-isatty v0.0.14
+	github.com/longhorn/backupstore v0.0.0-20230505020448-631a9fe919eb
+	github.com/longhorn/longhorn-manager v1.4.1
```

Comment:
should we keep v1.4.2 instead?

Reply:
yap, change back to v1.4.2. Thanks for the reminder!